content/5_Plat/Sweep_Line.mdx

```diff
@@ -55,6 +55,86 @@ To achieve the desired $\mathcal{O}(n)$ complexity per layer, we need to be able
 
 Because each step now runs in linear time and there is a total of $\lceil \log n \rceil$ steps, by the [master theorem](https://en.wikipedia.org/wiki/Master_theorem_(analysis_of_algorithms)) our solution now runs in $\mathcal{O}(n\log n)$.
 
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+struct Point {
+	long double x, y;
+	bool operator<(const Point &other) {
+		if (x == other.x) return y < other.y;
+		return x < other.x;
+	}
+};
+
+const pair<Point, Point> INF{{-1e9, -1e9}, {1e9, 1e9}};
+long double dist(pair<Point, Point> a) {
+	long double d1 = a.first.x - a.second.x;
+	long double d2 = a.first.y - a.second.y;
+	return sqrt(d1 * d1 + d2 * d2);
+}
+
+pair<Point, Point> get_closest_points(pair<Point, Point> a,
+                                      pair<Point, Point> b) {
+	return (dist(a) < dist(b) ? a : b);
+}
+
+pair<Point, Point> strip_solve(
+    vector<Point> &points) {  // Brute force for points with near mid_point
```

Comment:
```suggestion
/** Brute force for points with near mid_point */
pair<Point, Point> strip_solve(vector<Point> &points) {
```

Reply:
Done. I think the `pre-commit.ci `bot keeps changing this.